thewalrus/fock_gradients.py

```diff
@@ -23,36 +23,72 @@
 .. autosummary::
     :toctree: api
 
-    Xgate
```

Comment:
(How come the Xgate was removed? I suppose the user can always call `Dgate(f(x))`)

Reply:
There is no gain in keeping a separate Dgate for real displacement (=Xgate).